Description: desktop-entry-lacks-main-category
 The categories for this desktop entry do not contain any Main 
 Categories, only Additional Categories.
Author: Ruben Molina <rmolina@udea.edu.co>
Forwarded: not-needed
Last-Update: 2011-06-25

Index: kst-2.0.3/src/kst/kst.desktop
===================================================================
--- kst-2.0.3.orig/src/kst/kst.desktop
+++ kst-2.0.3/src/kst/kst.desktop
@@ -4,7 +4,7 @@ Exec=kst2 -caption "%c" %i %m
 Icon=kst
 DocPath=kst/index.html
 MimeType=application/x-kst;
-Categories=Qt;KDE;Science;
+Categories=Graphics;Qt;KDE;Science;
 Name=Kst
 Name[hi]=केएसटी
 Name[ta]=கேஎஸ்டி
